mitodl/mitxonline
# Updates button styling to sync border widths; updates close button on drawer

#### Pre-Flight checklist


#### What are the relevant tickets?

Closes #1292 
Closes #1291 

#### What's this PR do?

Updates the styling on the Enroll Now button so the border width doesn't change when you hover into and out of the button.
Updates the close widget to be more like the design.

#### How should this be manually tested?

Hovering into and out of the Enroll Now button in the Program Drawer should no longer cause the UI to move about. 
The close button should be a larger thin X now instead of the `&times;` glyph, and the program title should no longer run into it if the title is sufficiently long.

#### Screenshots (if appropriate)

![CleanShot 2022-12-19 at 11 45 46 2](https://user-images.githubusercontent.com/945611/208491613-bd8baeaa-42e7-4046-b6a1-6e163937edd0.gif)
![Screen Shot 2022-12-19 at 11 45 21 AM](https://user-images.githubusercontent.com/945611/208491617-40f3e88f-a13f-480e-a30c-5ab1dfbec403.png)
![image](https://user-images.githubusercontent.com/945611/208491702-ca51506f-7d31-4ec1-9a8b-c30859aa1bf6.png)
